TestApp/app/screens/TransmissionScreen.js

```diff
@@ -18,14 +20,53 @@ export default class TransmissionScreen extends Component {
     tabBarIcon: () => <Image style={{ width: 24, height: 24 }} source={TransmissionTabBarIcon} />
   }
 
+  targetTokensProperties = targetTokens.reduce((map, el) => {
+    map[el.key] = [];
+    return map;
+  }, {});
+
   state = {
-    targetToken: targetTokens[0]
+    targetToken: targetTokens[0],
+    showProperties: true,
+    properties: this.targetTokensProperties[targetTokens[0].key]
   }
 
   async componentWillMount() {
     await AppCenter.startFromLibrary(Analytics);
   }
 
+  async addProperty(property) {
+    const target = await Analytics.getTransmissionTarget(this.state.targetToken.key);
+    await target.propertyConfigurator.setEventProperty(property.name, property.value);
+    this.setState((state) => {
+      state.properties.push(property);
+      this.targetTokensProperties[this.state.targetToken.key] = state.properties;
+      return state;
+    });
+  }
+
+  async removeProperty(propertyName) {
+    const target = await Analytics.getTransmissionTarget(this.state.targetToken.key);
```

Comment:
Is inheritance coming in a separate PR?

Also it uses the new APIs but I see the PR targets develop and not `feature/transmission-target-properties-ui`, how did you test and could lint pass by calling non existing APIs?

Reply:
We've developed and tested this with API in #412, but divided in 2 PR. So, I think this should be merged only after #412 is merged.

Reply:
What about inheritance? Is this coming after? Today the app gets all targets at root level.